### PR TITLE
[luci/svc] Revise ShapeInfer_StridedSlice with int64

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise ShapeInfer_StridedSlice to follow original source with int64.

ONE-DCO-1.0-Signed-off-by: SaeHie Park <saehie.park@gmail.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
for  #11301 , from #11369

`uint32_t` was used to match `Dimension` in shape, but would be better to follow original source
and then use `uint32_t` in final step.

**Comment by seanshpark** [comment]
@jinevening PTAL

**Review by SlavikMIPT — DISMISSED** [review]
LGTM

File: compiler/luci/service/src/ShapeInfer_StridedSlice.cpp [hunk 1]
```diff
@@ -433,7 +433,7 @@ loco::TensorShape infer_output_shape(const CircleStridedSlice *node)
   for (uint32_t idx = 0; idx < shape_size; ++idx)
   {
     // reverse copy
-    output_shape.dim(idx) = output_shape_vector.at(shape_size - 1u - idx);
+    output_shape.dim(idx) = static_cast<uint32_t>(output_shape_vector.at(shape_size - 1u - idx));
```
Comment (jinevening) [inline compiler/luci/service/src/ShapeInfer_StridedSlice.cpp:436]: Does our SE tool accepts this pattern? I think downcasting would need range check?

**Comment by seanshpark** [comment]
@jinevening , @SlavikMIPT PTAL

**Review by jinevening — APPROVED** [review]
LGTM

**Comment by seanshpark** [comment]
@SlavikMIPT PTAL

**Comment by seanshpark** [comment]
Landing without @SlavikMIPT second approval; first approval was OK.

**Pull request merged**